docs/install-data-masking-plugin.md

```diff
@@ -0,0 +1,65 @@
+# Install and remove the data masking plugin
+
+This feature was implemented in *Percona Server for MySQL* version Percona Server for MySQL 8.0.17-8.
+
+The Percona Data Masking plugin is a free and Open Source implementation of the
+*MySQL*’s data masking plugin. Data Masking provides a set of functions to hide
+sensitive data with modified content.
+
+## Install the plugin
+
+The following command installs the plugin and the functions:
+
+```{.bash data-prompt="$"}
+INSTALL PLUGIN data_masking SONAME 'data_masking.so';
+CREATE FUNCTION gen_blocklist RETURNS STRING
```

Comment:
Our implementation doesn't reuire these create function calls, the functions are registered automatically.

Reply:
Removed.